bubble/bubble/Models/Bubble.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Bubble.swift
+//  bubble
+//
+//  Created by Siraj Zaneer on 2/8/18.
+//  Copyright © 2018 CS 408. All rights reserved.
+//
+
+import UIKit
+import Firebase
+
+class Bubble {
+
+    var text: String!
+    var owner: String!
+    var voteCount: Int!
+    var uid: String!
+    var geopoint: GeoPoint!
```

Comment:
Can you follow the pattern for models similar to the User class using private variables and getters? More info as to why this pattern is a good idea here: https://softwareengineering.stackexchange.com/questions/143736/why-do-we-need-private-variables

Reply:
Sounds good, only thing that I want to change though is since these variables should never be changed once set in the initializer I'll just make them read only public variables.